core/src/main/java/org/apache/accumulo/core/client/TableOfflineException.java

```diff
@@ -26,7 +27,7 @@ private static String getTableName(Instance instance, String tableId) {
     if (tableId == null)
       return " <unknown table> ";
     try {
-      String tableName = Tables.getTableName(instance, tableId);
+      String tableName = Tables.getTableName(instance, new Table.ID(tableId));
```

Comment:
> I think we should keep anything we do in this regard out of the public API

Yes, took me a moment to realize that this wasn't actually public facing. Still applies when viewed in a "it's a nice API to use"

> I imagine there was a good reason, but I don't have insight into the history of that part of KeyExtent and there don't appear to be any comments explaining why it was needed.

I know it was/is a big problem for them. Seeing the prevalence of this change, I assume it would also be non-trivial for us..